enginetest/evaluation.go

```diff
@@ -362,14 +363,15 @@ func injectBindVarsAndPrepare(
 ) (string, map[string]*querypb.BindVariable, error) {
 	parsed, err := sqlparser.Parse(q)
 	if err != nil {
-		return q, nil, err
+		return q, nil, sql.ErrSyntaxError.New(err)
```

Comment:
Why is this wrapping necessary here?

Reply:
Some error tests expected this `errors.Kind`. I could also change the tests